official/utils/flags/_benchmark.py

```diff
@@ -43,6 +43,12 @@ def define_benchmark(benchmark_log_dir=True, bigquery_uploader=True):
       help=help_wrap("The type of benchmark logger to use. Defaults to using "
                      "BaseBenchmarkLogger which logs to STDOUT. Different "
                      "loggers will require other flags to be able to work."))
+  flags.DEFINE_string(
+      name="benchmark_test_id", short_name="bti", default=None,
+      help=help_wrap("The unique test ID of the benchmark run. It could be the "
```

Comment:
Maybe a note that it is designed for human consumption? That is, nothing depends on this within the system.

Reply:
Done.